release/_bom.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+function download_bnd_files {
+	lc_cd "${_BUNDLES_DIR}/deploy"
```

Comment:
@pgydev, can you please move this code to be inside the `if` below?

As fair as I could understand, we just need to use it only if we need to perform the download, am I right?

If so, how about it do the following?
```
lc_download "https://repo1.maven.org/maven2/biz/aQute/bnd/biz.aQute.remote.agent/6.4.0/biz.aQute.remote.agent-6.4.0.jar" "${_BUNDLES_DIR}/deploy/biz.aQute.remote.agent-6.4.0.jar"
```